autoPyTorch/api/base_task.py

```diff
@@ -651,6 +655,13 @@ def _do_traditional_prediction(self, num_run: int, time_left: int, func_eval_tim
                     if status == StatusType.SUCCESS:
                         self._logger.info(
                             f"Fitting {cls} took {runtime}s, performance:{cost}/{additional_info}")
+                        configuration = additional_info['pipeline_configuration']
+                        origin = additional_info['configuration_origin']
+                        del additional_info['configuration_origin']
```

Comment:
I think you know it, but this `del` does not change the memory usage unless you re-define `configuration` or `origin` at some points.
```
In [1]: d = {}

In [2]: d[0] = [1, 2, 3, 4]

In [3]: a = d[0]

In [4]: id(a) == id(d[0])
Out[4]: True

In [5]: a[2] = 6

In [6]: id(a) == id(d[0])
Out[6]: True

In [7]: a = []

In [8]: id(a) == id(d[0])
Out[8]: False
```

Reply:
actually, I did not know this. The idea was also not to reduce memory usage. But I think I can remove these lines as the intended purpose is no longer valid